maskrcnn_benchmark/utils/tensorboard.py

```diff
@@ -0,0 +1,19 @@
+import time
+from datetime import datetime
+
+
+def get_tensorboard_writer(local_rank, distributed):
+    try:
+        from tensorboardX import SummaryWriter
+    except ImportError:
+        raise ImportError(
+            'To use tensorboard please install tensorboardX '
+            '[ pip install tensorflow tensorboardX ].'
+        )
+
+    if not distributed or (distributed and local_rank == 0):
```

Comment:
Can't you simplify this condition to something like
```
if distributed_rank == 0:
    ...
```
?

Also, it's safer to use the global `distributed_rank` instead of `local_rank`, as `local_rank` can be zero for each machine.

Reply:
I didn't know that variable existed 👍

Reply:
is it better to add `output_dir` as parameter to `get_tensorboard_writer` func?

Reply:
yes, I'm doing that already. will push it soon